docs/spec.md

```diff
@@ -4106,11 +4106,11 @@ The following diagram shows how elements in `results[k]` are computed from
 
 ![](images/spec/reduce_window.svg)
 
-More formally, `results[:][result_index] = reduce(windows, init_values, axes(inputs[:]), body)` where:
+More formally, `results[:][result_index] = reduce(window, init_values, axes(inputs[:]), body)` where:
 
   * `padded_inputs = pad(inputs[:], init_values[:], padding[:, 0], padding[:, 1], base_dilations)`.
   * `window_start = result_index * window_strides`.
-  * `windows = slice(padded_inputs[:], window_start, window_start + window_dimensions, window_dilations)`.
+  * `window = slice(padded_inputs[:], window_start, window_start + window_dimensions, window_dilations)`.
```

Comment:
There can be multiple inputs, so I think it is reasonable to say that there are multiple windows?